docs/translations/translations.json

```diff
@@ -68,11 +68,14 @@
   "feedbackCommentLabel": "Comment",
   "feedbackFailed": "Couldn't submit feedback. Please try again later.",
   "feedbackMessage": "Was this page helpful?",
-  "feedbackMessageDown": "Please let us know what we could do to improve this page.",
-  "feedbackMessageUp": "Please let us know what we should keep doing more of.",
+  "feedbackMessageDown": "Please let us know what we could do to improve this page (optional)",
+  "feedbackMessageUp": "Please let us know what we should keep doing more of (optional)",
```

Comment:
```suggestion
  "feedbackMessageUp": "Thanks! What did you like about this page? (optional)",
```

Reply:
Maybe we should remove the "Thanks" to keep the idea that the answer is not sent yet

Reply:
That makes sense @alexfauquette